public/assets/scss/_contact.scss

```diff
@@ -0,0 +1,157 @@
+.contact-holding {
+    background-image: url('../img/backgrounds/faq-holding.png');
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+    justify-content: center;
+    height: 25rem;
+    width: 100%;
+
+    &-heading {
```

Comment:
The exact classes are used for the FAQ page. Instead of recreating these classes for each page, I think it is better to have them as an individual class so that future edits will be easier. https://github.com/Highrachy/ballers-ws/blob/a8a2d0e56ff83c4c0104fc6a7453356c0b99ea1c/public/assets/scss/_faq.scss#L9-L19

Reply:
fonts designs aren't consistent across pages

Reply:
I agree with you that most of the fonts styles are not consistent, however, it is better to reuse classes for the ones that are consistent. Can you update the styles, please?